src/OpenLoco/Scenario.cpp

```diff
@@ -255,7 +254,7 @@ namespace OpenLoco::Scenario
 
         setDayProgression(0);
 
-        _scenario_ticks = 0;
+        ScenarioManager::setScenarioTicks(0);
```

Comment:
I take it ticks 1 is reset on scenario start and ticks 2 is set by the scenario editor.

Reply:
i'm not sure about ticks2, it's only ever incremented/decremented, it isn't actually set anywhere like ticks1 is